test/smoke/insight.js

```diff
@@ -1,4 +1,24 @@
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
+const fetch = require('node-fetch');
+
 describe('Insight', () => {
-  // /insight-api-dash/blocks?limit=1
-  it('should return block');
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+    describe(nodeName, () => {
+      it('insight should return block',
+        async () => {
+          const response = await fetch(`http://${networkConfig.inventory._meta.hostvars[nodeName].public_ip}:3001/insight-api-dash/blocks?limit=1`);
+          const { blocks } = await response.json();
+          expect(blocks).to.have.lengthOf(1);
+        });
+
+      it('should return master node list', async () => {
```

Comment:
Why do we need this test?

Reply:
why not? isight should sync data from all master nodes. do you think it's extra verification? I can remove it. btw, now it failed due to AssertionError: expected [] to have a length of 13 but got 0

Reply:
'Coz here we just test that insight is working. We don't test that masternode list working properly in dash core

Reply:
removed the test completely